### ISSUE TITLE
Failing on - Ensure PHP packages are installed

### ISSUE THREAD

**Issue by joz-d7** [geerlingguy/drupal-vm#2136]
### Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 2.2.14
VirtualBox 6.1.16r140961
ansible 2.9.6
  config file = None
  configured module search path = ['/Users/joz/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/local/Cellar/ansible/2.9.6_1/libexec/lib/python3.8/site-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.8.2 (default, Mar 11 2020, 00:29:50) [Clang 11.0.0 (clang-1100.0.33.17)]
```

### Your OS

 - **macOS - BigSur** 

### Full console output

https://gist.github.com/joz-d7/695d2cd20d6f4451884e6c0112d7a1b8

### Summary

I am unable to run vagrant up due to errors in PHP dependencies, I suspect it is a permission issue earlier in the pipeline.
Inside the VM I get the following error when trying to run `apt-get update` (Probably not related)

```vagrant@local:~$ apt-get update
Reading package lists... Done
W: Unable to read /etc/apt/sources.list.d/packagecloud_io_varnishcache_varnish64_ubuntu.list - open (13: Permission denied)
W: Unable to read /etc/apt/sources.list.d/ppa_ondrej_php_bionic.list - open (13: Permission denied)
E: Could not open lock file /var/lib/apt/lists/lock - open (13: Permission denied)
E: Unable to lock directory /var/lib/apt/lists/
W: Problem unlinking the file /var/cache/apt/pkgcache.bin - RemoveCaches (13: Permission denied)
W: Problem unlinking the file /var/cache/apt/srcpkgcache.bin - RemoveCaches (13: Permission denied)```

**Comment by joz-d7** [comment]
Following is a list of the plugins I am using (probably not related)
`
vagrant plugin list
vagrant-auto_network (1.0.3, global)
  - Version Constraint: > 0
vagrant-cachier (1.2.1, global)
  - Version Constraint: > 0
vagrant-hostsupdater (1.2.1, global)
  - Version Constraint: > 0
vagrant-vbguest (0.29.0, global)
  - Version Constraint: > 0
`

and I am using **Ubuntu 16.04 LTS ** OS for the VM

**Comment by joz-d7** [comment]
Solved by removing the 2.0.10 and 2.0.9 box then pulling a new box in

```
vagrant box remove geerlingguy/drupal-vm --box-version 2.0.10
vagrant box remove geerlingguy/drupal-vm --box-version 2.0.9
vagrant box add geerlingguy/drupal-vm --box-version 2.0.9
```
